webrtc.html

```diff
@@ -6404,6 +6407,11 @@ <h2>
                     <var>attr</var> in «{{candidate}}, {{sdpMid}},
                     {{sdpMLineIndex}}, {{usernameFragment}}»:
                       <ol>
+                        <p class="note">
```

Comment:
a `p` can't be a direct child of `ol` - it should either be integrated in the first `li`, or moved to a different place altogether